playground/update.sh

```diff
@@ -1,24 +1,39 @@
-#! /bin/sh
+#!/bin/bash
 set -e
 
 go install github.com/gopherjs/gopherjs/...
 
 go generate github.com/gopherjs/gopherjs.github.io/playground/internal/imports
 
-gopherjs install -m github.com/gopherjs/gopherjs.github.io/playground github.com/gopherjs/gopherjs/js
-cp $GOPATH/bin/playground.js $GOPATH/src/github.com/gopherjs/gopherjs.github.io/playground/playground.js
-cp $GOPATH/bin/playground.js.map $GOPATH/src/github.com/gopherjs/gopherjs.github.io/playground/playground.js.map
+# Build playground itself.
+gopherjs build -m
 
-PKG=$GOPATH/src/github.com/gopherjs/gopherjs.github.io/playground/pkg
+# The GOPATH workspace where the GopherJS project is.
+GOPHERJSGOPATH=`go list -f '{{.Root}}' github.com/gopherjs/gopherjs`
```

Comment:
I suggest using `$()` instead of backticks. Backticks aren't technically deprecated, though style guidelines usually suggest sticking to `$()`.

Same for line 24 (which you didn't touch in this PR)